pythoncalculator/multiply.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```

Comment:
Hi @tiaxlauren looks good just a typo here I think its adding rather than multiply.

```suggestion
    return x * y
```

Cheers,
Eve

Reply:
Great spot! Thank you